cmd/notary/tuf.go

```diff
@@ -83,6 +83,12 @@ var cmdTUFVerifyTemplate = usageTemplate{
 	Long:  "Verifies if the data passed in STDIN is included in the remote trusted collection identified by the Globally Unique Name.",
 }
 
+var cmdTUFDeleteTemplate = usageTemplate{
+	Use:   "delete [ GUN ]",
+	Short: "Deletes all content for a trusted collection",
+	Long:  "Deletes all content for a trusted collection identified by the Globally Unique Name, can delete both local and remote data based on flags.",
```

Comment:
nitpick:  "Deletes all local content for a trusted collection identified by the Globally Unique Name.  Remote data can also be deleted based on flags." ?